src/aspire/utils/random.py

```diff
@@ -49,19 +49,28 @@ def randn(*args, **kwargs):
     with Random(seed):
         uniform = np.random.rand(*args, **kwargs)
         result = np.sqrt(2) * erfinv(2 * uniform - 1)
-        # TODO: Rearranging elements to get consistent behavior with MATLAB 'randn2'
-        result = m_reshape(result.flatten(), args)
+        # Note, rearranging elements to get consistent behavior with MATLAB 'randn2'
+        result = result.T.reshape(args, order="F")
+
         return result
 
 
 def rand(size, seed=None):
```

Comment:
Maybe this is a good time to rename this to `matlab_rand` or something so that people don't call it by mistake? At first glance, it's not clear what the difference is between `rand` and `random`.

Reply:
Sure! Done.